app/models/ncr/work_order.rb

```diff
@@ -119,7 +119,7 @@ def current_approver_email_address
       if self.pending?
         self.currently_awaiting_approvers.first.email_address
       else
-        self.approving_official.email_address
+        self.approving_official ? self.approving_official.email_address : self.system_approver_emails.first
```

Comment:
this ternary operator is hiding an `if/else` within an `if/else` - any chance we could move the logic ELSEwhere? (see what I did there -- ?? :100: )